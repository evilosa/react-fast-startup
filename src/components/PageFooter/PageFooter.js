import React from 'react'
import PropTypes from 'prop-types'

const PageFooter = () => {
  return <div>Page footer</div>
}

PageFooter.propTypes = {
  children: PropTypes.node
}

PageFooter.displayName = 'PageFooter'

export default PageFooter