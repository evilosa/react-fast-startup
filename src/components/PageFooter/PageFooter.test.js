import React from 'react'
import { shallow } from 'enzyme'
import PageFooter from './PageFooter'

describe('PageFooter', () => {
  const component = shallow(<PageFooter/>)

  it('renders correct', () => {
    expect(component).toMatchSnapshot()
  })
})