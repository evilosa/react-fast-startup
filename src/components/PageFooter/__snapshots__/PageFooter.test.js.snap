// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageFooter renders correct 1`] = `
<div>
  Page footer
</div>
`;
